app/services/offline_poll_presenter.rb

```diff
@@ -0,0 +1,28 @@
+class OfflinePollPresenter
+  def initialize(peerage)
+    @peerage = peerage
+    @candidates = Candidate.where_order(@peerage)
+	end
+  def vote_list
+    @candidates.where(vote:true).order(:sca_name).map { |c| OfflinePollCandidatePresenter.new(c) }
```

Comment:
extract method?

Reply:
I'm not going to change this now, but I will if I have a third kind of list or if I have to do this again in the future